### ISSUE TITLE
Add Ability To Select Where Backup File Goes

### ISSUE THREAD

**Issue by landry314** [ukanth/afwall#1169]
Creating a folder in the sdcard just for a backup file is not good form. It should either be put in the Android/data/ folder automatically or even better, the user can select the location when he is backing up or the user can select a default path in settings where he always wants the backup file to go.

I have a folder just for backup files and I'd love to set that instead of copying it there manually!

**Comment by SynapseScribe** [comment]
this should be higher priority, as you loose the backup upon reinstalling. i just lost all rules...

**Comment by landry314** [comment]
yeah, the backup file is in the /Android/data/dev.ukanth.ufirewall/files/ folder which is a pain in the ass. i wrote a script to move it. would love to just set a backup folder.

mv -v /sdcard/Android/data/dev.ukanth.ufirewall/files/* /sdcard/Backup/

**Comment by jgilmore** [comment]
Where the actual fuck is my backup file so I can transfer it to the new phone? I've been trying to figure how WHERRE THE JFUKCNIGN EHLLL YOUI IDIOTS PUT SHITHIS R OFVER ANO NOQ NOW!@!!@@Q@#Q

Damnit! This is so fucking stupid. 

Ive even got another app ( my keyfile!) that stores it in "content:L/bullshit/external storage/documents" but it's sure as fuck nowhere on the SD card. Or external storage for that matter. I can't share it, I can't find it even with root, and I can't access it, and the baby is crying so I have to give up for now. DAmnit.

Fix this!
